.firebaserc

```diff
@@ -2,8 +2,10 @@
   "projects": {
     "default": "dart-dev",
     "dev": "dartlang-org-dev",
-    "staging-0": "dartlang-org-staging-0",
-    "staging-1": "dartlang-org-staging-1",
+    "staging-0": "dart-dev-staging-0",
+    "staging-1": "dart-dev-staging-1",
+    "staging-0-old": "dartlang-org-staging-0",
+    "staging-1-old": "dartlang-org-staging-1",
```

Comment:
@kwalrath: are the `dartlang-org-staging-*` projects still active?

Reply:
Merging now. If these projects have been deleted, then I'll remove the corresponding lines in a followup PR.